daliuge-translator/dlg/dropmake/cwl.py

```diff
@@ -47,6 +50,13 @@ def create_workflow(drops, cwl_filename, buffer):
           Non-BashShellApp nodes are unable to be implemented in CWL
     """
 
+    # search the drops for non-BashShellApp drops,
+    # if found, the graph cannot be translated into CWL
+    for index, node in enumerate(drops):
+        dataType = node.get('dt', '')
+        if dataType in UNSUPPORTED_CATEGORIES:
```

Comment:
See the comment above.

Reply:
Thanks for the feedback. Since it is relatively minor, I'll just make the changes and merge, without asking you to re-review.